srcs/http/status_code.cpp

```diff
@@ -1,8 +1,11 @@
 #include "status_code.hpp"
+#include "utils.hpp"
 
 namespace http {
 
-ReasonPhrase InitReasonPhrase() {
+typedef std::map<EStatusCode, std::string> ReasonPhrase;
+
+ReasonPhrase StatusCode::InitReasonPhrase() {
```

Comment:
再度 typedef しなくてもこれで大丈夫そうです
```suggestion
StatusCode::ReasonPhrase StatusCode::InitReasonPhrase() {
```